dockertest.go

```diff
@@ -40,19 +41,27 @@ func (r *Resource) GetPort(id string) string {
 }
 
 // NewPool creates a new pool. You can pass an empty string to use the default, which is taken from the environment
-// variable DOCKER_URL or if that is not defined a sensible default for the operating system you are on.
+// variable DOCKER_URL, or from docker-machine if the environment variable DOCKER_MACHINE_NAME is set,
+// or if neither is defined a sensible default for the operating system you are on.
 func NewPool(endpoint string) (*Pool, error) {
+	var client *dc.Client
+	var err error
+
 	if endpoint == "" {
 		if os.Getenv("DOCKER_URL") != "" {
 			endpoint = os.Getenv("DOCKER_URL")
+		} else if os.Getenv("DOCKER_MACHINE_NAME") != "" {
```

Comment:
What do you mean by "per spec"? It typically 

You set the environment on your computer using `eval $(docker-machine env default)` when sets the following environment:
```
export DOCKER_TLS_VERIFY="1"
export DOCKER_HOST="tcp://192.168.174.132:2376"
export DOCKER_CERT_PATH="/Users/martin/.docker/machine/machines/default"
export DOCKER_MACHINE_NAME="default"
```